server/controllers/finance/reports/debtors/annual_clients_report.handlebars

```diff
@@ -16,39 +37,56 @@
           <tr class="text-capitalize text-center" style="background-color: #ddd;">
             <th>{{translate "TABLE.COLUMNS.ACCOUNT"}}</th>
             <th>{{translate "TABLE.COLUMNS.NAME"}}</th>
-            <th>{{translate "REPORT.OPENING_BALANCE"}}</th>
-            <th>{{translate "TABLE.COLUMNS.DEBIT"}}</th>
-            <th>{{translate "TABLE.COLUMNS.CREDIT"}}</th>
-            <th>{{translate "FORM.LABELS.DIFFERENCE"}}</th>
-            <th>{{translate "REPORT.CLOSING_BALANCE"}}</th>
+            {{#if cashClients}}
+            <th class="rotate"><div>{{translate "TABLE.COLUMNS.CASHQ"}}</div></th>
+            {{/if}}
+            {{#if showLockedColumn}}
+            <th class="rotate"><div>{{translate "TABLE.COLUMNS.LOCKEDQ"}}</div></th>
+            {{/if}}
+            <th class="right">{{translate "REPORT.OPENING_BALANCE"}}</th>
+            <th class="right">{{translate "TABLE.COLUMNS.DEBIT"}}</th>
+            <th class="right">{{translate "TABLE.COLUMNS.CREDIT"}}</th>
+            <th class="right">{{translate "FORM.LABELS.DIFFERENCE"}}</th>
+            <th class="right">{{translate "REPORT.CLOSING_BALANCE"}}</th>
           </tr>
         </thead>
         <tbody>
           <!-- print a row for each debtor account-->
           {{#each rows as | row |}}
             <tr>
               <td>{{row.accountNumber}}</td>
-              <td>{{row.groupName }}</td>
+              <td>{{row.groupName}}</td>
+              {{#if ../cashClients}}
+              <td class="center">{{#if row.cashClient}}{{else}}<strong>X</strong>{{/if}}</td>
```

Comment:
It took me a while to parse that this was an inversion - blank if `row.cashClient` and X otherwise.  Let's bring this naming in line with what it actually intends, which I relayed in a previous comment.